### PR TITLE
RATIS-1843. Add existing markdown documents to site.xml in the ratis-docs project

### PR DESCRIPTION
**Pull request by wzhallright**
## What changes were proposed in this pull request?

As a noob, I've been hoping for more documentation. I noticed some md files were merged but not added to site.xml.
According to [RATIS-1630](https://issues.apache.org/jira/browse/RATIS-1630)
Maybe we can add this markdown to site.xml so we can view the documentation locally.   
### after
image   
When i view the document locally. I found some formatting issues like below:    
image    
I fixed this at the issue:   
image

## What is the link to the Apache JIRA

[RATIS-1843](https://issues.apache.org/jira/browse/RATIS-1843)

## How was this patch tested?

(Please explain how this patch was tested. Ex: unit tests, manual tests)
(If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

### CODE REVIEW

**Comment by wzhallright** [comment]
@SzyWilliam @szetszwo Could you help review this when you have time? Thanks!

File: ratis-docs/src/site/site.xml [hunk 1]
```diff
@@ -35,6 +35,9 @@
             <item name="Getting Started" href="start/index.html"/>
             <item name="Concepts" href="concept/index.html"/>
             <item name="Ratis-shell" href="cli.html" />
+            <item name="Snapshot" href="snapshot.html" />
+            <item name="Metrics" href="metrics.html" />
+            <item name="Configurations" href="configurations.html" />
```
Comment (szetszwo) [inline ratis-docs/src/site/site.xml:40]: Could you sort them in alphabetical order?

**Review by szetszwo — COMMENTED** [review]
@wzhallright , thanks a lot for working on this!  Just a minor comment inlined.

**Review by SzyWilliam — APPROVED** [review]
+1 the changes look good to me. @wzhallright Thanks a lot for working on this!

File: ratis-docs/src/site/site.xml (hunk 1 above)
Comment (wzhallright) [reply ratis-docs/src/site/site.xml]: Sorted them.Please check again when you have time.Thanks.

**Review by szetszwo — COMMENTED** [review]
+1 the change looks good.

**Review by szetszwo — APPROVED** [review]

**Pull request merged**